probe/probe.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package probe
+
+
+import (
+	"fmt"
+	"syscall"
+    "strings"
+	"golang.org/x/mod/semver"
+    "io/ioutil"
+    "log"
+
+)
+
+func PrintProbeResult() error{
+    
+    fmt.Printf("Host: \n")
+    fmt.Printf("\t Observability/Audit: %s \n", checkHostAuditSupport())
+    fmt.Printf("\t Enforcement: %s \n", checkHostLsmSupport())
```

Comment:
Let's prettify the print, WDYT?
Like showing supported in Green, Partial in Orange maybe. Bold for the Host or something.

Perhaps can leverage https://github.com/fatih/color or some more suiting lib.